### ISSUE TITLE
Create API Endpoint for Showing the Team Member Details

### ISSUE THREAD

**Issue by deshraj** [Cloud-CV/EvalAI#679]
Objective:
- Create an API endpoint with URL `/api/web/teams` which shows all team members on `GET` request.

**Comment by aka-jain** [comment]
@deshraj Are we going to show team creation form somewhere?

**Comment by deshraj** [comment]
Oh my bad. Only `GET` request is allowed on this API. Updating the description. :)

**Comment by yadavankit** [comment]
@deshraj you can assign this to me :)

**Comment by yadavankit** [comment]
![screen shot 2017-02-17 at 12 00 11 am](https://cloud.githubusercontent.com/assets/8945824/23034920/6fa132b4-f4a3-11e6-966c-d30198f23cb8.png)
@aka-jain does this solve the issue?

**Comment by deshraj** [comment]
@yadavankit it would be great if you can integrate this with the `our-team` view. Let me know if you  cannot do this then we can open another issue for that. :-)

**Comment by yadavankit** [comment]
@deshraj ok will do 👍

**Comment by deshraj** [comment]
Thanks :)

**Comment by aka-jain** [comment]
Also @yadavankit member title is for short description which will use text editor in admin. 
@deshraj I think this response will be better, what say?
```{
  "core": [
    {
      "name": "Akash",
      "desc": "I Love <i class='fa fa-coffee'></i>" // text editor field,
      ...
    }
  ],
  "contributors": [
    {
      "name": "Akash",
      "desc": "I Love <i class='fa fa-coffee'></i>" // text editor field,
      ...
    }
  ]
}```

**Comment by deshraj** [comment]
Yes, you are correct. @yadavankit can you please take care of this situation?

**Comment by deshraj** [comment]
@aka-jain can we take care of this grouping of contributors and the core team at the client side using the `ng-if`?

**Comment by yadavankit** [comment]
@deshraj yeah...I was about to ask for it too

**Comment by deshraj** [comment]
Ok cool then. @yadavankit lets do it this way. :)

**Comment by aka-jain** [comment]
@deshraj No need to use ng if here, as we can store data in 2 different object, one is for core and other for contributors.

**Comment by deshraj** [comment]
Yeah @aka-jain, I agree with this approach. But I was more concerned about the fact that if the # of Types increases in future, then we should take care of that already.

**Comment by aka-jain** [comment]
Agree, let's go with @yadavankit design.

**Comment by deshraj** [comment]
Fixed in #682
